file_catalog/server.py

```diff
@@ -125,6 +125,10 @@ def create(config: Dict[str, Any],
         "config": config,
         "db": mongo,
         "debug": debug,
+        "routestats": {
```

Comment:
I think it might need to be `route_stats` in the config object?

`rest_tools/server/handler.py:        route_stats = defaultdict(functools.partial(RouteStats, **config['route_stats']))`

https://github.com/WIPACrepo/rest-tools/blob/master/rest_tools/server/handler.py#L78

Reply:
Sadly, you're right.  I must not have looked close enough.